matplotlib-tests/bld.bat

```diff
@@ -0,0 +1 @@
+xcopy lib\matplotlib\tests %SP_DIR%\matplotlib /S
```

Comment:
Sorry, I should have mentioned I haven't tested this.  I'll have to fire up my Windows partition (unless there's some automatic testing here I can take advantage of).